include/vapor/VolumeRenderer.h

```diff
@@ -58,10 +58,10 @@ class RENDER_API VolumeRenderer : public Renderer
     virtual std::string _getDefaultAlgorithmForGrid(const Grid *grid) const;
     bool _needToSetDefaultAlgorithm() const;
     
-    unsigned int _VAO = NULL;
-    unsigned int _VBO = NULL;
-    unsigned int _VAOChunked = NULL;
-    unsigned int _VBOChunked = NULL;
+    unsigned int _VAO = (int)NULL;
+    unsigned int _VBO = (int)NULL;
+    unsigned int _VAOChunked = (int)NULL;
+    unsigned int _VBOChunked = (int)NULL;
```

Comment:
1. I think we better give these `unsigned int` value of 0?

Reply:
These are handles that happen to be represented as integers. NULL indicates that they are not just numbers.